pkg/master/controller.go

```diff
@@ -83,6 +86,11 @@ func (c *Controller) Start() {
 	repairClusterIPs := servicecontroller.NewRepair(c.ServiceClusterIPInterval, c.ServiceRegistry, c.ServiceClusterIPRange, c.ServiceClusterIPRegistry)
 	repairNodePorts := portallocatorcontroller.NewRepair(c.ServiceNodePortInterval, c.ServiceRegistry, c.ServiceNodePortRange, c.ServiceNodePortRegistry)
 
+	// Initialize the master leases
+	if c.MasterLeases != nil {
```

Comment:
I think, at one point, there was a reason that involved not knowing what EndpointInterval was when you constructed the lease manager.  That's does not seem to be relevant any more, so I'll fix it.